docs/content/developers/building-contributing.md

```diff
@@ -24,70 +24,45 @@ There are several ways to use DDEV’s latest-committed HEAD version:
 
 Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) creates GitHub artifacts you can use for testing, so you can download the one you need from the PR page, install it locally, and test using that build.
 
-Download and unzip the appropriate binary and place it in your `$PATH`.
+!!!tip "You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev) (perform a rollback)."
 
-## Rollback to a previous version
+Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. Since not every OS has `$HOME/bin` in `$PATH`, you can create the folder and add it to your path by updating `~/.bashrc`, `~/.zshrc`, or another relevant shell configuration file with these commands:
 
-You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev).
-
-### Homebrew with macOS or Linux
-
-If you’re using Homebrew, start by unlinking your current binary:
-
-```
-brew unlink ddev
+```bash
+mkdir ~/bin
+echo 'export PATH="$HOME/bin:$PATH"' >>~/.bashrc
+source ~/.bashrc
+# Verify that `$HOME/bin` is the first entry in your `$PATH`
+echo $PATH
 ```
 
-Next, unzip the binary you downloaded, make it executable, and move it to your bin folder:
+Choose a ZIP file for your OS and architecture, download it using `wget`, `curl`, `dl` or using a browser and unzip it, make it executable, and move it to the `~/bin` folder:
```

Comment:
Thinking more about this, the reason it irked me when I hovered over the links, was that I saw `[...]/2246879499.zip` which are "automagically" transformed.

I am not a fan of magic with computers, but in the end, clicking seems to be the easiest method.

It would work better for me, if there was a basic explanation of what it looks like, and what happens -- then I have no problem recommending the click-based solution. Something like this, recommending the click method, by moving it first and clarifying how it works:

```suggestion
Download a ZIP file for your OS and architecture by clicking the link (the result is something like `ddev-macos-arm64.zip`) or using `wget`, `curl`, `dl` and unzip it, make it executable, and move it to the `~/bin` folder:
```